Shared/Services/Backups/BackupsService.swift

```diff
@@ -486,6 +486,130 @@ class BackupsService: ObservableObject {
             })
 
     }
+    
+    func downloadFolderBackup(device: Device, downloadAt: URL, folderId: String) async throws {
```

Comment:
if the below method is called `downloadBackupFile`, let's call this `downloadBackupFolder`, try to be consistent with the namings as it reduces cognitive load and wtf moments. Or you can call the other one `downloadFileBackup`, like you did in the XPCBackupService protocol, just be consistent

Reply:
sorry is my bad!